### DISCUSSION TITLE
Network Error: Unexpected token '<', "<!DOCTYPE "... is not valid JSON

### DISCUSSION THREAD

**Discussion by arcndz** [requarks/wiki#6602] [Error / Bug Report] [answered]
Hello everyone,

I set up wiki js with PostgreSQL and Apache and it's working. However, I kept getting this error message intermittently while working on the wiki (creating pages, adding users in the administration area, etc.) 

Network Error: Unexpected token '<', "<!DOCTYPE "... is not valid JSON

Any idea what could be the issue?

**Comment by NGPixel** [comment, marked as answer]
You have a reverse proxy blocking requests (e.g. Cloudflare, nginx, etc.).

**Reply by arcndz** [reply]
Yes I am using Apache/2.4.37
How can I stop this from happening? Is there a configuration that I can use so that the reverse proxy will not block requests?

**Reply by NGPixel** [reply]
There's no special config needed. A normal config wouldn't block anything...

**Reply by arcndz** [reply]
Does wiki js support HTTP to HTTPS redirection with a custom self-signed certificate? I used Apache for this purpose to remove the port number at the end of the URL and also secure it to HTTPS

**Reply by NGPixel** [reply]
https://docs.requarks.io/install/config#https
